README.md

```diff
@@ -1,119 +1,20 @@
-# ParseUI
+# [ARCHIVE] ParseUI
 
-[![Build Status](http://img.shields.io/travis/ParsePlatform/ParseUI-iOS/master.svg?style=flat)](https://travis-ci.org/ParsePlatform/ParseUI-iOS)
-[![Pod Version](http://img.shields.io/cocoapods/v/ParseUI.svg?style=flat)](http://cocoadocs.org/docsets/ParseUI/)
-[![Pod Platform](http://img.shields.io/cocoapods/p/ParseUI.svg?style=flat)](http://cocoadocs.org/docsets/ParseUI/)
-[![Pod License](http://img.shields.io/cocoapods/l/ParseUI.svg?style=flat)](https://github.com/ParsePlatform/ParseUI-iOS/blob/master/LICENSE)
-
-## Overview
-
-`ParseUI` is a collection of a handy user interface components to be used with Parse iOS SDK,  
-which streamline and simplify logging in / signing up `PFUser`s and displaying a list of `PFObject`s.
+# ParseUI has moved to [Parse-SDK-iOS-OSX](https://github.com/parse-community/Parse-SDK-iOS-OSX)
```

Comment:
Could retain the original README contents as well under a README_ARCHIVED.md or something like that? Just in case we need to reference any of the old material. Although I imagine you might be planning to add this to the iOS/OSX readme?

Reply:
That's a good idea, I'll re-add the other readmes in the repository (in the project subfolder). What do you think?